mujoco_warp/_src/types.py

```diff
@@ -1260,6 +1261,7 @@ class Model:
   mat_rgba: wp.array2d(dtype=wp.vec4)
   geompair2hfgeompair: wp.array(dtype=int)  # warp only
   block_dim: BlockDim  # warp only
+  collision_geom_pair: tuple  # warp only
```

Comment:
I think you can do `tuple[tuple[int, int], ...]` or some such?

Also maybe we should make it clearer these are geom types?  Maybe name it `geom_type_pair`?